src/authn/ccloudProvider.test.ts

```diff
@@ -174,11 +173,9 @@ describe("ConfluentCloudAuthProvider", () => {
     assert.ok(handleSessionRemovedStub.calledWith(true));
   });
 
-  it("handleSessionCreated() should update the provider's internal state, fire the _onDidChangeSessions event, and adjust polling", async () => {
+  it("handleSessionCreated() should update the provider's internal state, fire the _onDidChangeSessions event.", async () => {
     const storageManager = getStorageManager();
     const setSecretStub = sandbox.stub(storageManager, "setSecret").resolves();
-    const pollStartStub = sandbox.stub(pollCCloudConnectionAuth, "start").resolves();
```

Comment:
No more pollCCloudConnectionAuth interval poller, therefore no more turning it on and off based on if ccloud connection exists and is auth'd.